(Android)(Mali)(OpenGL) Armored Core III Portable screen flickering required dual source blending

Game Info
![screenshot_2018-09-29-15-43-38](https://user-images.githubusercontent.com/37603562/46242779-1f3a9100-c3bc-11e8-97fa-7f8bae264926.png)

Game sometimes ok :/
![screenshot_2018-09-29-15-42-50](https://user-images.githubusercontent.com/37603562/46242789-34172480-c3bc-11e8-8f9c-c47849704c8a.png)
But always has a darkoverlay?
![screenshot_2018-09-29-15-42-41](https://user-images.githubusercontent.com/37603562/46242794-4a24e500-c3bc-11e8-942d-5741053232f2.png)

PPSSPP use v1.6.3-510-gbd678d4f4
(Im using default settings only.)

Phone Specs
Octacore CPU
GPU - Mali-450 MP 
Android 5.1 Lollipop

Why I can't download the ge dump on this game using latest ppsspp apk on buildbot.

maybe related to scoop storage?

Waiting for a game to start...
Debugger could not connect
Debugger could not connect
Debugger connected
Bad message: invalid JSON
34:11:792 UNKNOWN GAME E[IO]: Format/JSONReader.h:76 Error at (1): unquoted key
"event


Bad message: invalid JSON
34:11:793 UNKNOWN GAME E[IO]: Format/JSONReader.h:76 Error at (1): unquoted key
"event


Bad message: invalid JSON
34:11:794 UNKNOWN GAME E[IO]: Format/JSONReader.h:76 Error at (1): unquoted key
"event


Bad message: invalid JSON
34:11:832 UNKNOWN GAME E[IO]: Format/JSONReader.h:76 Error at (1): unquoted key
"event


34:13:848 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:1170 Got insets: 100:0:0:0
34:13:851 UNKNOWN GAME I[G3D]: UI/NativeApp.cpp:238 g_Config.screenRotation = 1
34:13:854 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:791 NativeApp.resume() - resuming audio
34:13:880 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:1170 Got insets: 100:0:0:0
34:13:886 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:13:886 UNKNOWN GAME I[G3D]: jni/app-android.cpp:852 NativeApp.displayInit() restoring
34:13:886 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:313 EmuThreadStop - stopping (displayInit)...
34:13:886 UNKNOWN GAME I[G3D]: jni/app-android.cpp:856 BeginAndroidShutdown. Looping until emu thread done...
34:13:895 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 9297.48ms (+9264.11ms) / 0.1fps
Kernel processing time: 3.06 ms
Slowest syscall: sceGeListUpdateStallAddr : 1.83 ms
Most active syscall: sceGeListUpdateStallAddr : 1.84 ms
DL processing time: 1.84 ms
Draw calls: 543, flushes 443, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8723 0 0
Vertices: 31674 cached: 0 uncached: 91986
FBOs active: 2 (evaluations: 0)
Textures: 150, dec: 0, invalidated: 0, hashed: 1751 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1547687 (48.312084 per vertex)
Vertex, Fragment, Programs loaded: 14, 16, 20

34:13:896 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:291 QUIT_REQUESTED found, left loop. Setting state to STOPPED.
34:13:896 UNKNOWN GAME I[G3D]: GLES/GPU_GLES.cpp:298 GPU_GLES: DeviceLost
34:13:896 UNKNOWN GAME I[G3D]: Common/TextureCacheCommon.cpp:1689 Texture cached cleared from 150 textures
34:13:896 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Destroying FBO for 04088000 : 480 x 272 x 3
34:13:896 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Destroying FBO for 04000000 : 480 x 272 x 3
34:13:896 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:1004 NativeShutdownGraphics
34:14:016 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:1044 NativeShutdownGraphics done
34:14:016 UNKNOWN GAME I[G3D]: jni/app-android.cpp:861 Joining emu thread
34:14:016 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:266 GL submission thread paused. Frame=0
34:14:016 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:300 Leaving emu thread
34:14:016 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:320 EmuThreadJoin - joined
34:14:017 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:174 ThreadEnd
34:14:017 UNKNOWN GAME I[G3D]: jni/AndroidJavaGLContext.cpp:26 AndroidJavaEGLGraphicsContext::Shutdown
34:14:017 UNKNOWN GAME I[G3D]: jni/app-android.cpp:869 Shut down both threads. Now let's bring it up again!
34:14:017 UNKNOWN GAME I[G3D]: jni/AndroidJavaGLContext.cpp:13 AndroidJavaEGLGraphicsContext::InitFromRenderThread
34:14:017 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:304 EmuThreadStart
34:14:017 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:134 Updating inflight frames to 2
34:14:017 UNKNOWN GAME I[G3D]: jni/app-android.cpp:891 Restored.
34:14:017 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:270 Entering emu thread
34:14:017 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:279 Runloop: Graphics context available! 0xb40000762c7664c0
34:14:017 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:885 NativeInitGraphics
34:14:017 UNKNOWN GAME I[G3D]: Text/draw_text_android.cpp:30 Initializing TextDrawerAndroid with DPI scale 0.545455
34:14:018 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:954 NativeInitGraphics completed
34:14:018 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:283 Graphics initialized. Entering loop.
34:14:019 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 307.47ms (+290.78ms) / 3.3fps
Kernel processing time: 0.13 ms
Slowest syscall: sceKernelWaitSema : 0.05 ms
Most active syscall: sceKernelWaitSema : 0.06 ms
DL processing time: 0.00 ms
Draw calls: 0, flushes 0, clears 0 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 0 0 0 0
Vertices: 0 cached: 0 uncached: 0
FBOs active: 0 (evaluations: 0)
Textures: 0, dec: 0, invalidated: 0, hashed: 0 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 0 (0.000000 per vertex)
Vertex, Fragment, Programs loaded: 0, 0, 0

34:14:079 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:237 Running first frame (0)
34:14:080 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:3 Creating FBO for 04000000 (z: 04110000) : 480 x 272 x 3
34:14:102 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:469 Creating 1440 x 816 FBO using DEPTH24_STENCIL8 texture
34:14:134 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:456 Creating 480 x 272 FBO using no depth
34:14:145 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 217.88ms (+201.19ms) / 4.6fps
Kernel processing time: 6.39 ms
Slowest syscall: sceGeListUpdateStallAddr : 5.66 ms
Most active syscall: sceGeListUpdateStallAddr : 5.67 ms
DL processing time: 5.66 ms
Draw calls: 550, flushes 450, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8951 0 0
Vertices: 31695 cached: 0 uncached: 92007
FBOs active: 1 (evaluations: 79)
Textures: 79, dec: 93, invalidated: 0, hashed: 1752 kB
Readbacks: 1, uploads: 1
GPU cycles executed: 1548563 (48.293327 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:147 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 93.26ms (+76.58ms) / 10.7fps
Kernel processing time: 0.07 ms
Slowest syscall: __sceSasCore : 0.01 ms
Most active syscall: __sceSasCore : 0.04 ms
DL processing time: 0.00 ms
Draw calls: 0, flushes 0, clears 0 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 0 0 0 0
Vertices: 0 cached: 0 uncached: 0
FBOs active: 1 (evaluations: 0)
Textures: 79, dec: 0, invalidated: 0, hashed: 0 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 0 (0.000000 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:150 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:3 Creating FBO for 04088000 (z: 04110000) : 480 x 272 x 3
34:14:153 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:936 NativeApp.backbufferResize(2340 x 1080)
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:928 RecalcDPI: display_xres=2340 display_yres=1080
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:929 RecalcDPI: g_dpi=440.000000 g_dpi_scale_x=0.545455 g_dpi_scale_y=0.545455
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:930 RecalcDPI: dp_xscale=0.545299 dp_yscale=0.545370
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:931 RecalcDPI: dp_xres=1276 dp_yres=589
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:932 RecalcDPI: pixel_xres=2340 pixel_yres=1080
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:952 NativeApp::backbufferResize: Size didn't change.
34:14:153 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:14:164 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:469 Creating 1440 x 816 FBO using DEPTH24_STENCIL8 texture
34:14:169 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:456 Creating 480 x 272 FBO using no depth
34:14:176 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:14:178 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 108.21ms (+91.52ms) / 9.2fps
Kernel processing time: 3.63 ms
Slowest syscall: sceGeListUpdateStallAddr : 3.44 ms
Most active syscall: sceGeListUpdateStallAddr : 3.45 ms
DL processing time: 3.44 ms
Draw calls: 550, flushes 450, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8952 0 0
Vertices: 31695 cached: 0 uncached: 92007
FBOs active: 2 (evaluations: 2)
Textures: 81, dec: 2, invalidated: 0, hashed: 1748 kB
Readbacks: 1, uploads: 1
GPU cycles executed: 1548565 (48.293327 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:186 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 98.92ms (+65.55ms) / 10.1fps
Kernel processing time: 2.78 ms
Slowest syscall: sceGeListUpdateStallAddr : 2.17 ms
Most active syscall: sceGeListUpdateStallAddr : 2.18 ms
DL processing time: 2.17 ms
Draw calls: 551, flushes 451, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8998 0 0
Vertices: 31675 cached: 0 uncached: 91941
FBOs active: 2 (evaluations: 2)
Textures: 83, dec: 2, invalidated: 0, hashed: 1748 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1548257 (48.311192 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:199 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 78.72ms (+45.35ms) / 12.7fps
Kernel processing time: 5.27 ms
Slowest syscall: sceGeListUpdateStallAddr : 4.68 ms
Most active syscall: sceGeListUpdateStallAddr : 4.69 ms
DL processing time: 4.68 ms
Draw calls: 551, flushes 451, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8997 0 0
Vertices: 31675 cached: 0 uncached: 91941
FBOs active: 2 (evaluations: 2)
Textures: 85, dec: 2, invalidated: 0, hashed: 1748 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1548255 (48.311192 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:208 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:14:211 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 57.17ms (+23.80ms) / 17.5fps
Kernel processing time: 3.80 ms
Slowest syscall: sceGeListUpdateStallAddr : 3.22 ms
Most active syscall: sceGeListUpdateStallAddr : 3.23 ms
DL processing time: 3.23 ms
Draw calls: 551, flushes 451, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 9003 0 0
Vertices: 31651 cached: 0 uncached: 91869
FBOs active: 2 (evaluations: 2)
Textures: 87, dec: 2, invalidated: 0, hashed: 1752 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1547787 (48.332661 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:221 idle0        I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:14:233 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 45.85ms (+12.48ms) / 21.8fps
Kernel processing time: 6.18 ms
Slowest syscall: sceGeListUpdateStallAddr : 4.81 ms
Most active syscall: sceGeListUpdateStallAddr : 4.83 ms
DL processing time: 4.81 ms
Draw calls: 570, flushes 470, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 9550 0 0
Vertices: 32127 cached: 0 uncached: 93183
FBOs active: 2 (evaluations: 8)
Textures: 95, dec: 8, invalidated: 0, hashed: 1766 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1558401 (47.912876 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:233 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Decimating FBO for 04000000 (480 x 272 x 3), age 6
34:14:287 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Decimating FBO for 04088000 (480 x 272 x 3), age 6
34:14:688 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 33.79ms (+0.43ms) / 29.6fps
Kernel processing time: 20.69 ms
Slowest syscall: sceGeListUpdateStallAddr : 16.10 ms
Most active syscall: sceGeListUpdateStallAddr : 16.11 ms
DL processing time: 16.10 ms
Draw calls: 533, flushes 433, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8391 0 0
Vertices: 31931 cached: 0 uncached: 92817
FBOs active: 2 (evaluations: 2)
Textures: 141, dec: 2, invalidated: 0, hashed: 1736 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1552163 (48.084213 per vertex)
Vertex, Fragment, Programs loaded: 14, 15, 19

34:14:819 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:14:839 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:15:917 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:798 NativeApp.pause() - pausing audio
34:15:953 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:15:971 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:35:536 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:1170 Got insets: 100:0:0:0
34:35:541 UNKNOWN GAME I[G3D]: UI/NativeApp.cpp:238 g_Config.screenRotation = 1
34:35:545 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:791 NativeApp.resume() - resuming audio
34:35:574 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:1170 Got insets: 100:0:0:0
34:35:580 UNKNOWN GAME I[G3D]: jni/app-android.cpp:852 NativeApp.displayInit() restoring
34:35:580 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:313 EmuThreadStop - stopping (displayInit)...
34:35:581 UNKNOWN GAME I[G3D]: jni/app-android.cpp:856 BeginAndroidShutdown. Looping until emu thread done...
34:35:590 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:35:595 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 19605.87ms (+19572.51ms) / 0.1fps
Kernel processing time: 5.71 ms
Slowest syscall: sceGeListUpdateStallAddr : 4.39 ms
Most active syscall: sceGeListUpdateStallAddr : 4.41 ms
DL processing time: 4.40 ms
Draw calls: 523, flushes 423, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8069 0 0
Vertices: 31650 cached: 0 uncached: 92034
FBOs active: 2 (evaluations: 0)
Textures: 157, dec: 0, invalidated: 0, hashed: 1712 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1545899 (48.333553 per vertex)
Vertex, Fragment, Programs loaded: 14, 16, 20

34:35:595 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:291 QUIT_REQUESTED found, left loop. Setting state to STOPPED.
34:35:595 UNKNOWN GAME I[G3D]: GLES/GPU_GLES.cpp:298 GPU_GLES: DeviceLost
34:35:595 UNKNOWN GAME I[G3D]: Common/TextureCacheCommon.cpp:1689 Texture cached cleared from 157 textures
34:35:596 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Destroying FBO for 04000000 : 480 x 272 x 3
34:35:596 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Destroying FBO for 04088000 : 480 x 272 x 3
34:35:596 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:1004 NativeShutdownGraphics
34:35:596 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:1044 NativeShutdownGraphics done
34:35:596 UNKNOWN GAME I[G3D]: jni/app-android.cpp:861 Joining emu thread
34:35:597 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:266 GL submission thread paused. Frame=0
34:35:597 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:300 Leaving emu thread
34:35:597 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:320 EmuThreadJoin - joined
34:35:597 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:174 ThreadEnd
34:35:597 UNKNOWN GAME I[G3D]: jni/AndroidJavaGLContext.cpp:26 AndroidJavaEGLGraphicsContext::Shutdown
34:35:597 UNKNOWN GAME I[G3D]: jni/app-android.cpp:869 Shut down both threads. Now let's bring it up again!
34:35:597 UNKNOWN GAME I[G3D]: jni/AndroidJavaGLContext.cpp:13 AndroidJavaEGLGraphicsContext::InitFromRenderThread
34:35:597 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:304 EmuThreadStart
34:35:598 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:134 Updating inflight frames to 2
34:35:598 UNKNOWN GAME I[G3D]: jni/app-android.cpp:891 Restored.
34:35:599 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:270 Entering emu thread
34:35:599 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:279 Runloop: Graphics context available! 0xb40000762c7664c0
34:35:599 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:885 NativeInitGraphics
34:35:599 UNKNOWN GAME I[G3D]: Text/draw_text_android.cpp:30 Initializing TextDrawerAndroid with DPI scale 0.545455
34:35:599 UNKNOWN GAME I[SYSTEM]: UI/NativeApp.cpp:954 NativeInitGraphics completed
34:35:599 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:283 Graphics initialized. Entering loop.
34:35:601 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 190.09ms (+173.41ms) / 5.3fps
Kernel processing time: 0.10 ms
Slowest syscall: __sceSasCore : 0.01 ms
Most active syscall: __sceSasCore : 0.04 ms
DL processing time: 0.00 ms
Draw calls: 0, flushes 0, clears 0 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 0 0 0 0
Vertices: 0 cached: 0 uncached: 0
FBOs active: 0 (evaluations: 0)
Textures: 0, dec: 0, invalidated: 0, hashed: 0 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 0 (0.000000 per vertex)
Vertex, Fragment, Programs loaded: 0, 0, 0

34:35:664 UNKNOWN GAME I[G3D]: OpenGL/GLRenderManager.cpp:237 Running first frame (0)
34:35:664 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:3 Creating FBO for 04000000 (z: 04110000) : 480 x 272 x 3
34:35:688 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:469 Creating 1440 x 816 FBO using DEPTH24_STENCIL8 texture
34:35:723 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:456 Creating 480 x 272 FBO using no depth
34:35:733 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 223.98ms (+207.30ms) / 4.5fps
Kernel processing time: 6.02 ms
Slowest syscall: sceGeListUpdateStallAddr : 5.31 ms
Most active syscall: sceGeListUpdateStallAddr : 5.33 ms
DL processing time: 5.32 ms
Draw calls: 521, flushes 421, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8052 0 0
Vertices: 31525 cached: 0 uncached: 91671
FBOs active: 1 (evaluations: 77)
Textures: 77, dec: 91, invalidated: 0, hashed: 1717 kB
Readbacks: 1, uploads: 1
GPU cycles executed: 1543365 (48.445900 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:735 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 93.54ms (+76.86ms) / 10.7fps
Kernel processing time: 0.20 ms
Slowest syscall: sceDisplayWaitVblankStart : 0.05 ms
Most active syscall: __sceSasCore : 0.10 ms
DL processing time: 0.00 ms
Draw calls: 0, flushes 0, clears 0 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 0 0 0 0
Vertices: 0 cached: 0 uncached: 0
FBOs active: 1 (evaluations: 0)
Textures: 77, dec: 0, invalidated: 0, hashed: 0 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 0 (0.000000 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:740 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:3 Creating FBO for 04088000 (z: 04110000) : 480 x 272 x 3
34:35:743 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:936 NativeApp.backbufferResize(2340 x 1080)
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:928 RecalcDPI: display_xres=2340 display_yres=1080
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:929 RecalcDPI: g_dpi=440.000000 g_dpi_scale_x=0.545455 g_dpi_scale_y=0.545455
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:930 RecalcDPI: dp_xscale=0.545299 dp_yscale=0.545370
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:931 RecalcDPI: dp_xres=1276 dp_yres=589
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:932 RecalcDPI: pixel_xres=2340 pixel_yres=1080
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:952 NativeApp::backbufferResize: Size didn't change.
34:35:743 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:35:756 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:469 Creating 1440 x 816 FBO using DEPTH24_STENCIL8 texture
34:35:761 UNKNOWN GAME I[G3D]: OpenGL/GLQueueRunner.cpp:456 Creating 480 x 272 FBO using no depth
34:35:767 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:35:780 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 16.68ms / 59.9fps, actual = 121.58ms (+104.90ms) / 8.2fps
Kernel processing time: 5.33 ms
Slowest syscall: sceGeListUpdateStallAddr : 4.99 ms
Most active syscall: sceGeListUpdateStallAddr : 5.00 ms
DL processing time: 4.99 ms
Draw calls: 519, flushes 419, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 7999 0 0
Vertices: 31475 cached: 0 uncached: 91533
FBOs active: 2 (evaluations: 2)
Textures: 79, dec: 2, invalidated: 0, hashed: 1714 kB
Readbacks: 1, uploads: 1
GPU cycles executed: 1542259 (48.491089 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:789 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 100.75ms (+67.38ms) / 9.9fps
Kernel processing time: 4.86 ms
Slowest syscall: sceGeListUpdateStallAddr : 2.75 ms
Most active syscall: sceGeListUpdateStallAddr : 2.76 ms
DL processing time: 2.75 ms
Draw calls: 524, flushes 424, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8154 0 0
Vertices: 31495 cached: 0 uncached: 91563
FBOs active: 2 (evaluations: 2)
Textures: 81, dec: 2, invalidated: 0, hashed: 1714 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1542969 (48.472996 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:798 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:35:814 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 92.87ms (+59.51ms) / 10.8fps
Kernel processing time: 5.96 ms
Slowest syscall: sceGeListUpdateStallAddr : 4.80 ms
Most active syscall: sceGeListUpdateStallAddr : 4.82 ms
DL processing time: 4.80 ms
Draw calls: 524, flushes 424, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8154 0 0
Vertices: 31495 cached: 0 uncached: 91563
FBOs active: 2 (evaluations: 2)
Textures: 83, dec: 2, invalidated: 0, hashed: 1714 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1542969 (48.472996 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:823 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 67.80ms (+34.43ms) / 14.8fps
Kernel processing time: 2.58 ms
Slowest syscall: sceGeListUpdateStallAddr : 2.27 ms
Most active syscall: sceGeListUpdateStallAddr : 2.28 ms
DL processing time: 2.28 ms
Draw calls: 529, flushes 429, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8309 0 0
Vertices: 31515 cached: 0 uncached: 91593
FBOs active: 2 (evaluations: 2)
Textures: 85, dec: 2, invalidated: 0, hashed: 1714 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1543679 (48.454926 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:828 idle0        I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:35:836 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 48.13ms (+14.76ms) / 20.8fps
Kernel processing time: 3.55 ms
Slowest syscall: sceGeListUpdateStallAddr : 2.47 ms
Most active syscall: sceGeListUpdateStallAddr : 2.48 ms
DL processing time: 2.48 ms
Draw calls: 529, flushes 429, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 8309 0 0
Vertices: 31515 cached: 0 uncached: 91593
FBOs active: 2 (evaluations: 2)
Textures: 87, dec: 2, invalidated: 0, hashed: 1714 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1543679 (48.454926 per vertex)
Vertex, Fragment, Programs loaded: 14, 14, 18

34:35:836 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Decimating FBO for 04000000 (480 x 272 x 3), age 6
34:35:888 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Decimating FBO for 04088000 (480 x 272 x 3), age 6
34:36:360 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 38.28ms (+4.92ms) / 26.1fps
Kernel processing time: 10.20 ms
Slowest syscall: sceGeListUpdateStallAddr : 7.80 ms
Most active syscall: sceGeListUpdateStallAddr : 7.81 ms
DL processing time: 7.80 ms
Draw calls: 636, flushes 536, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 11541 0 0
Vertices: 32238 cached: 0 uncached: 93120
FBOs active: 2 (evaluations: 1)
Textures: 124, dec: 1, invalidated: 0, hashed: 1810 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1564603 (47.816769 per vertex)
Vertex, Fragment, Programs loaded: 14, 16, 20

34:36:525 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:36:575 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
34:40:693 UNKNOWN GAME N[SCEDISP]: HLE/sceDisplay.cpp:530 Dropping frames - budget = 33.37ms / 30.0fps, actual = 33.69ms (+0.32ms) / 29.7fps
Kernel processing time: 10.46 ms
Slowest syscall: sceGeListUpdateStallAddr : 7.70 ms
Most active syscall: sceGeListUpdateStallAddr : 7.73 ms
DL processing time: 7.71 ms
Draw calls: 569, flushes 469, clears 1 (cached: 0)
Num Tracked Vertex Arrays: 0
Commands per call level: 2 9424 0 0
Vertices: 32336 cached: 0 uncached: 93816
FBOs active: 2 (evaluations: 0)
Textures: 170, dec: 0, invalidated: 0, hashed: 1752 kB
Readbacks: 0, uploads: 0
GPU cycles executed: 1562329 (47.732464 per vertex)
Vertex, Fragment, Programs loaded: 14, 17, 21

34:55:076 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1812 Block transfer readback 11x5 4bpp from 04088000 (x:173 y:114 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:077 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:3 Creating FBO for 04154000 (z: 04110000) : 128 x 64 x 3
34:55:077 UNKNOWN GAME W[SCEGE]: Common/FramebufferManagerCommon.cpp:413 FBO reusing depthbuffer, c=04154000/d=04110000 and c=04000000/d=04110000
34:55:108 idle0        I[G3D]: OpenGL/GLQueueRunner.cpp:469 Creating 384 x 192 FBO using DEPTH24_STENCIL8 texture
34:55:111 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:170 y:114 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:144 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:168 y:114 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:177 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:165 y:115 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:211 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:163 y:116 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:245 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:161 y:117 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:278 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:160 y:117 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:311 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:158 y:118 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:345 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:157 y:119 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:378 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:155 y:120 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:412 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:153 y:121 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:444 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:152 y:123 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:479 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:150 y:124 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:512 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:148 y:125 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:548 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:147 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:587 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:145 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:613 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:144 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:780 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:144 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:814 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:145 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:847 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:145 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:881 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:146 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:912 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:148 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:949 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:149 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:55:981 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:150 y:127 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:014 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:152 y:128 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:048 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:154 y:129 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:081 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:157 y:129 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:114 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:160 y:130 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:145 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:162 y:130 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:181 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:165 y:129 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:213 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:168 y:128 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:248 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:171 y:126 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:280 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:174 y:124 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:314 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:177 y:122 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:348 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:180 y:120 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:380 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04000000 (x:183 y:118 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:416 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:186 y:116 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:447 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:189 y:114 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:481 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:192 y:111 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:514 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:194 y:109 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:548 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:197 y:107 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:581 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:200 y:104 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:614 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:202 y:102 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:648 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:205 y:100 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:680 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:207 y:98 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:714 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:209 y:97 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:746 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:210 y:95 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:782 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:210 y:95 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:815 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04088000 (x:210 y:94 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:56:848 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 13x6 4bpp from 04000000 (x:211 y:94 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:076 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Decimating FBO for 04154000 (128 x 64 x 3), age 6
34:57:383 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1812 Block transfer readback 12x6 4bpp from 04000000 (x:206 y:121 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:383 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:3 Creating FBO for 04154000 (z: 04110000) : 128 x 64 x 3
34:57:383 UNKNOWN GAME W[SCEGE]: Common/FramebufferManagerCommon.cpp:413 FBO reusing depthbuffer, c=04154000/d=04110000 and c=04000000/d=04110000
34:57:410 idle0        I[G3D]: OpenGL/GLQueueRunner.cpp:469 Creating 384 x 192 FBO using DEPTH24_STENCIL8 texture
34:57:416 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 12x6 4bpp from 04088000 (x:206 y:121 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:449 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:206 y:122 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:483 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:204 y:122 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:514 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:203 y:122 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:547 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:201 y:123 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:582 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:199 y:123 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:616 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:197 y:124 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:652 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:195 y:125 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:682 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:193 y:126 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:717 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:191 y:126 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:748 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:189 y:125 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:783 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:187 y:125 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:815 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:185 y:124 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:851 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04000000 (x:183 y:122 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:881 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 11x5 4bpp from 04088000 (x:182 y:121 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:915 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:180 y:119 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:949 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:179 y:118 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:57:982 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:178 y:116 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:017 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:176 y:114 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:048 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:175 y:113 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:083 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:174 y:111 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:117 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:173 y:110 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:149 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:172 y:108 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:184 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:170 y:107 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:218 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:169 y:105 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:251 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:168 y:104 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:282 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:167 y:103 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:318 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:165 y:102 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:349 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:164 y:102 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:382 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:163 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:416 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:162 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:451 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:161 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:484 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:160 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:514 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04000000 (x:159 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:550 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 10x5 4bpp from 04088000 (x:159 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:582 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:159 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:618 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:158 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:649 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:158 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:685 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:157 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:717 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:157 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:749 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:157 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:784 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:157 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:816 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:156 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:852 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:156 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:883 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:156 y:100 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:918 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:156 y:100 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:950 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:156 y:100 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:58:983 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:156 y:100 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:59:017 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:156 y:100 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:59:048 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04000000 (x:156 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
34:59:088 UNKNOWN GAME W[G3D]: Common/FramebufferManagerCommon.cpp:1796 Inter-buffer block transfer 9x4 4bpp from 04088000 (x:156 y:101 stride:512) -> 04154000 (x:0 y:0 stride:128)
35:00:079 UNKNOWN GAME I[FRAMEBUF]: Common/FramebufferManagerCommon.cpp:1 Decimating FBO for 04154000 (128 x 64 x 3), age 6
35:00:393 UNKNOWN GAME I[SYSTEM]: jni/app-android.cpp:798 NativeApp.pause() - pausing audio
35:00:447 CSoundAtrac3 I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)
35:00:476 UNKNOWN GAME I[G3D]: jni/app-android.cpp:1241 NativeApp.setDisplayParameters(2340 x 1080, dpi=440, refresh=60.00)

```
34:11:792 UNKNOWN GAME E[IO]: Format/JSONReader.h:76 Error at (1): unquoted key
"event
```

Interesting, it's like you're getting a truncated message or something.  I wonder what could be causing this...

-[Unknown]

--- a/assets/compat.ini
+++ b/assets/compat.ini
@@ -136,7 +136,7 @@ ULJM08069 = true
 NPJH50625 = true
 
 [DepthRangeHack]
-# Phantasy Star Portable 2 and Infinity both use viewport depth outside [0, 1].
+# Phantasy Star Portable 2 and Infinity as well as some FromSoftware titles both use viewport depth outside [0, 1].
 # This gets clamped in our current implementation, but attempts to fix it run into
 # Other bugs, so we've restored this hack for now.
 # Phantasy Star Portable
@@ -163,6 +163,40 @@ ULJM91018 = true
 NPJH90062 = true
 # Phantasy Star Portable Infinity Demo
 NPJH90157 = true  # Infinity demo
+# Armored Core 3 Portable
+NPEH00045 = true
+NPUH10023 = true
+NPUH90059 = true
+UCAS40268 = true
+ULJM05492 = true
+ULKS46210 = true
+# Armored Core: Last Raven Portable
+NPEH00046 = true
+NPUH10024 = true
+NPUH90083 = true
+UCAS40302 = true
+UCKS45148 = true
+ULJM05611 = true
+# Armored Core: Silent Line Portable
+NPEH00047 = true
+NPJH90075 = true
+NPUH10025 = true
+NPUH90068 = true
+UCAS40289 = true
+ULJM05552 = true
+# MonHun Nikki
+NPJH50294 = true
+ULAS42285 = true
+ULJM05710 = true
+ULJM05881 = true
+ULJM08039 = true
+# SSX on Tour
+NPHH00081 = true
+ULES00192 = true
+ULJM05063 = true
+ULJM05389 = true
+ULKS46038 = true
+ULUS10042 = true
 
 [ClearToRAM]
 # SOCOM Navy Seals games require this. See issue #8973.

